src/pixelflow/core.py

```diff
@@ -42,8 +42,8 @@ def wrapper(x: npt.NDArray, y: npt.NDArray) -> Callable:
 
 
 @dataclasses.dataclass
-class PixelflowResult:
-    """Result container with addional `reduce` functionality."""
+class PixelflowResult():
```

Comment:
```suggestion
class PixelflowResult:
```

I don't think you need the parenthesis here